pkg/subctl/cmd/export.go

```diff
@@ -64,28 +66,47 @@ func exportService(cmd *cobra.Command, args []string) {
 
 	exitOnError("Error connecting to the target cluster", err)
 
-	_, clientSet, err := getClients(restConfig)
-	exitOnError("Error connecting to the target cluster", err)
-
-	lhClientSet, err := lighthouseClientset.NewForConfig(restConfig)
+	dynClient, clientSet, err := getClients(restConfig)
 	exitOnError("Error connecting to the target cluster", err)
+	restMapper, err := autil.BuildRestMapper(restConfig)
 
+	exitOnError("Error creating RestMapper for ServiceExport", err)
 	if serviceNamespace == "" {
 		if serviceNamespace, _, err = clientConfig.Namespace(); err != nil {
 			serviceNamespace = "default"
 		}
 	}
+
+	err = mcsv1a1.AddToScheme(scheme.Scheme)
+	exitOnError("Failed to add to scheme", err)
+
 	svcName := args[0]
+
 	_, err = clientSet.CoreV1().Services(serviceNamespace).Get(svcName, metav1.GetOptions{})
 	exitOnError(fmt.Sprintf("Unable to find the Service %q in namespace %q", svcName, serviceNamespace), err)
 
+	/* NOTE:
+	Lighthouse agent requires the LH ServiceExport installed for now. Operator generates
+	CRDs based on types in vendor directory. So we need to import lighthouse/apis
+	to force it to vendor it and create CRD for installation.
+	This will be removed in next release once we fully drop lighthouse CRDs.
+	*/
```

Comment:
Instead of leaving this is in the code path, you can just create an unused global var:

`var _ = v2alpha1.ServiceExport{}`

Reply:
Done